src/Core/Translation/Storage/Finder/TranslationFinder.php

```diff
@@ -70,11 +70,11 @@ private function removeTrailingLocaleFromDomains(MessageCatalogueInterface $cata
         $messages = $catalogue->all();
         $locale = $catalogue->getLocale();
         $localeSuffix = '.' . $locale;
-        $suffixLength = strlen($localeSuffix);
+        $suffixLength = strlen($localeSuffix) * -1;
```

Comment:
Can you explain why you did this?

Reply:
Can you add a little comment about why you added it.
I finally understand it's because of substr function, but the variable name is confusing, "My suffix length is -5" is quite distributing.
The old way was a little bit clear because everyone knows that having a minus in the substr function tells "Start from the end". But having a variable with "Something length", you're thinking about positive numbers